tools/lib/provision.py

```diff
@@ -115,6 +120,7 @@
     "trusty": "9.3",
     "xenial": "9.5",
```

Comment:
postgresql 10 has perf improvements over postgresql 9.x. Maybe 10 should be the default for all distros?

Reply:
An outdated beta benchmark: https://www.phoronix.com/scan.php?page=news_item&px=PostgreSQL-10-Beta-Benchmarks.

Reply:
We use the version of postgres shipped with each distro without any funny backports business.  This is the correct behavior unless we're willing to do a bunch of work to add PPAs and migrate data in production.

Reply:
OK, though for centos7's case, I looked at the [PGroonga install guide](https://pgroonga.github.io/install/centos.html) where it assumes postgres 10. The default system package is 9.2, but it sounds too outdated.

Reply:
Hmm, yeah, 9.2 is ancient.  I think as our strategy for EL, I'm fine with us just installing 10 and scripting everything to use that.